sdcm/cluster_k8s/__init__.py

```diff
@@ -1923,12 +1923,38 @@ def add_scylla_cluster_value(self, path: str, element: Any):
         )
 
     @property
-    def scylla_config_map(self):
+    @contextlib.contextmanager
+    def scylla_config_map(self) -> dict:
+        exists = False
+        config_map = None
```

Comment:
This line (config_map = None) and 1936 can be removed if you add "config_map = {}" to the except block on current line 1935

Reply:
addressed